clang/lib/AST/Interp/ByteCodeExprGen.cpp

```diff
@@ -1642,9 +1642,9 @@ bool ByteCodeExprGen<Emitter>::visitZeroInitializer(QualType QT,
   case PT_Uint64:
     return this->emitZeroUint64(E);
   case PT_IntAP:
+    return this->emitZeroIntAP(128, E); // FIXME: Ctx.getBitWidth()
   case PT_IntAPS:
-    assert(false);
-    return false;
+    return this->emitZeroIntAPS(128, E); // FIXME: Ctx.getBitWidth()
```

Comment:
Is something preventing us from supporting `getBitWidth()` right off the bat?

Reply:
Not anymore!